spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -296,7 +298,8 @@ protected void doStop() {
 
 				}
 				if (getClientManager() == null) {
-					this.mqttClient.disconnectForcibly(getDisconnectCompletionTimeout());
+					this.mqttClient.disconnectForcibly(ClientManager.QUIESCENT_TIMEOUT, getDisconnectCompletionTimeout(),
```

Comment:
Did you mean `getQuiescentTimeout()` as you do in the `MqttPahoMessageDrivenChannelAdapter` for MQTTv3?

Reply:
DOH!